src/languages/en-US.ts

```diff
@@ -559,6 +559,13 @@ export default class extends Language {
 		SETTINGS_STARBOARD_CHANNEL: 'The starboard channel. If you star a message, it will be posted there. Using the `setStarboardEmoji` command allows the emoji customization.',
 		SETTINGS_STARBOARD_IGNORECHANNELS: 'The channels I should ignore when listening for new stars.',
 		SETTINGS_STARBOARD_MINIMUM: 'The minimum amount of stars required before a message is posted to the starboard channel.',
+		SETTINGS_SUGGESTIONS_CHANNEL: 'The channel where suggestions will be sent',
+		SETTINGS_SUGGESTIONS_EMOJIS_UPVOTE: 'The upvote emoji Skyra reacts with on every suggestion',
+		SETTINGS_SUGGESTIONS_EMOJIS_DOWNVOTE: 'The downvote emoji Skyra reacts with on every suggestion',
+		SETTINGS_SUGGESTIONS_ON_ACTION_DM: 'If this setting is enabled, Skyra will DM the suggestion\'s author every time it is updated',
+		SETTINGS_SUGGESTIONS_ON_ACTION_REPOST: 'If this setting is enabled, Skyra will repost the suggestion\'s message every time it is updated. If it is disabled, it will edit the original message',
+		SETTINGS_SUGGESTIONS_ON_ACTION_HIDE_AUTHOR: 'This setting allows you to update suggestions anonymously. It will substitute the updater\'s name with either `An administrator` or `A moderator`, according to their permission level',
```

Comment:
```suggestion
		SETTINGS_SUGGESTIONS_CHANNEL: 'The channel where suggestions will be sent.',
		SETTINGS_SUGGESTIONS_EMOJIS_UPVOTE: 'The upvote emoji Skyra reacts with on every suggestion.',
		SETTINGS_SUGGESTIONS_EMOJIS_DOWNVOTE: 'The downvote emoji Skyra reacts with on every suggestion.',
		SETTINGS_SUGGESTIONS_ON_ACTION_DM: 'If this setting is enabled, Skyra will DM the suggestion\'s author every time it is updated.',
		SETTINGS_SUGGESTIONS_ON_ACTION_REPOST: 'If this setting is enabled, Skyra will repost the suggestion\'s message every time it is updated. If it is disabled, it will edit the original message.',
		SETTINGS_SUGGESTIONS_ON_ACTION_HIDE_AUTHOR: 'This setting allows you to update suggestions anonymously. It will substitute the updater\'s name with either `An administrator` or `A moderator`, according to their permission level.',
```